vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -156,22 +157,12 @@ func validateCreateOptions(r volume.Request, fsMap map[string]string) error {
 		r.Options[fsTypeTag] = fs.FstypeDefault
```

Comment:
Not really, the volume driver (this file) knows of volumes and it relies on the FS module to handle all of the format part. Which can include the selection of a default filesystem to use. The FS driver is the one thats keeping data on what file system types are available on the docker host anyway. In fact why set the fstype in the options here. When the format is done, the FS module can check that the option isn't present and select the default. The volume driver really has no idea what the default is anyway and no way to configure the default either by the plugin.